src/doc.c

```diff
@@ -822,16 +873,22 @@ void emitComment(Dsymbol *s, Scope *sc)
             }
             dc->pmacrotable = &sc->module->macrotable;
 
+            OutBuffer *buf = sc->docbuf;
+
             buf->writestring(ddoc_decl_s);
-            size_t o = buf->offset;
-            toDocBuffer(ad, buf, sc);
-            highlightCode(sc, ad, buf, o);
-            sc->lastoffset = buf->offset;
+            {
+                size_t o = buf->offset;
+                toDocBuffer(ad, buf, sc);
+                highlightCode(sc, ad, buf, o);
+                sc->lastoffset = buf->offset;
+            }
             buf->writestring(ddoc_decl_e);
 
             buf->writestring(ddoc_decl_dd_s);
-            dc->writeSections(sc, ad, buf);
-            emitMemberComments(ad, sc);
+            {
```

Comment:
same question (I understand when scoping is used for defining locals inside, but some of these do not)